### PR TITLE
Corrected duplicate 'by grouping' in comment.

### PR DESCRIPTION
**Pull request by cdpetersen**

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! 
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Corrected duplicate 'by grouping' in comment. 
  [cdpetersen](https://github.com/cdpetersen)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,21 +11,24 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
...
-* Hash scope suffixes if they are over 50 characters to prevent file paths from being too long. 
+* Hash scope suffixes if they are over 50 characters to prevent file paths from being too long.
   [Danielle Tomlinson](https://github.com/dantoml)
   [#5491](https://github.com/CocoaPods/CocoaPods/issues/5491)
 
 * Fix codesigning identity on watchOS and tvOS targets.    
   [Danielle Tomlinson](https://github.com/dantoml)
   [#5686](https://github.com/CocoaPods/CocoaPods/issues/5686)
 
+  * Corrected duplicate 'by grouping' in comment.
```
Comment (segiddins) [inline CHANGELOG.md]: this doesn't require a changelog entry

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,21 +11,24 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
...
-  
+
 * Improve performance of dependency resolution.
   [yanzhiwei147](https://github.com/yanzhiwei147)
   [#5510](https://github.com/CocoaPods/CocoaPods/pull/5510)
 
 ##### Bug Fixes
 
-* Hash scope suffixes if they are over 50 characters to prevent file paths from being too long. 
+* Hash scope suffixes if they are over 50 characters to prevent file paths from being too long.
```
Comment (segiddins) [inline CHANGELOG.md]: this should have 2 trailing spaces

**Comment by cdpetersen** [comment]
Removed unnecessary CHANGELOG changes.

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! 
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Corrected duplicate 'by grouping' in comment. 
  [cdpetersen](https://github.com/cdpetersen)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by segiddins** [comment]
Thanks!

**Pull request merged**